java/core/src/java/org/apache/orc/impl/DictionaryUtils.java

```diff
@@ -42,4 +44,29 @@ public static void getTextInternal(Text result, int position, DynamicIntArray ke
     }
     byteArray.setText(result, offset, length);
   }
+
+  /**
+   * Write a UTF8 string from the byteArray, using the offset in index-array,
+   * into an OutputStream
+   *
+   * @param out the output stream
+   * @param position position in the keyOffsets
+   * @param keyOffsets starting offset of the key (in byte) in the byte array
+   * @param byteArray storing raw bytes of all keys seen in dictionary
+   * @return the number of bytes written to the output stream
+   * @throw IOException if an I/O error occurs
```

Comment:
Unfortunately, this breaks our `Javadoc` generation. Here is the fix.
- https://github.com/apache/orc/pull/867